packages/server/src/service/sendinblue.ts

```diff
@@ -0,0 +1,62 @@
+import { processEnvOrThrow } from '../common'
+import { ContactsApi, ContactsApiApiKeys } from 'sib-api-v3-typescript'
+
+// Configure Authentication & API
+
+const key = processEnvOrThrow('SENDINBLUE_API_KEY')
+
+/**
+ * Tries to get SENDINBLUE_LIST_ID from process.env, if not present
+ * it defaults to 2
+ */
+const listId = () => {
+  try {
+    const rawEnv = processEnvOrThrow('SENDINBLUE_LIST_ID')
+    return Number.parseInt(rawEnv, 10)
+  } catch(e) {
+    return 2
+  }
+}
+
+class Sendinblue {
```

Comment:
Created a class here in case we need more Sendinblue functionality in the future, for example removing subscribers, adding names to our subscribers, etc.